tests/unit/column-picker.spec.ts

```diff
@@ -70,4 +70,26 @@ describe('Column Picker', () => {
     expect(wrapper.vm.$data.column).toEqual('columnB');
     expect(store.state.vqb.selectedColumns).toEqual(['columnB']);
   });
+
+  it('should not update step when selectedColumn is changed if prop "sync" is false', async () => {
+    const store: Store<any> = setupMockStore({
```

Comment:
I don't think we need to add the `Store<any>` annotation here.

Reply:
Need to make typescript happy :)
No quick fix in mind. Maybe we could try to propose something more elegant in another PR ?